### DISCUSSION TITLE
Can the value of Menu.Item be optional?

### DISCUSSION THREAD

**Discussion by s97712** [chakra-ui/ark#2710] [Q&A]
Why `value` is required? I trigger different actions through Menu.Item's onClick, and in most scenarios, value is not useful. I really don’t want to set unique value for each menu item.

**Comment by segunadebayo** [comment]
No, it can't be optional since we need it for keyboard navigation.

You can create a wrapper for `MenuItem` that uses `useId` under the hood.

**Comment by segunadebayo** [comment]
Closing — `value` is required by design for item identification.
